utilities/ImpactUtilities.groovy

```diff
@@ -109,8 +109,12 @@ def createImpactBuildList() {
 							// impactedFile found, but on Exclude List
 							//   Possible reasons: Exclude of file was defined after building the collection.
 							//   Rescan/Rebuild Collection to synchronize it with defined build scope.
-							if (props.verbose) println "!! $impactFile is impacted by changed file $changedFile, but is on Exlude List. Not added to build list."
+							if (props.verbose) println "*!* $impactFile is impacted by changed file $changedFile, but is on Exlude List. Not added to build list."
```

Comment:
If I'm not wrong, the error messages start with `*!` and not `*!*`, can you change it to be consistent with other scripts?